atlasdb-workload-server/src/main/java/com/palantir/atlasdb/workload/config/WorkloadServerInstallConfiguration.java

```diff
@@ -28,13 +26,7 @@
 import com.palantir.atlasdb.workload.workflow.WriteOnceDeleteOnceWorkflowConfiguration;
 import com.palantir.atlasdb.workload.workflow.bank.BankBalanceWorkflowConfiguration;
 import com.palantir.atlasdb.workload.workflow.ring.RingWorkflowConfiguration;
-import com.palantir.logsafe.DoNotLog;
-import org.immutables.value.Value;
 
-@DoNotLog
-@JsonDeserialize(as = ImmutableWorkloadServerInstallConfiguration.class)
-@JsonSerialize(as = ImmutableWorkloadServerInstallConfiguration.class)
-@Value.Immutable
 public interface WorkloadServerInstallConfiguration {
```

Comment:
I don't think internally you should be extending this, I'm assuming this is why you removed the annotations here.
Just take this as a field in the service config.

Reply:
I want to be able to do this:
```
@Value.Immutable
@ImmutablesConfigStyle
@JsonDeserialize(as = ImmutableWcWorkloadServerRuntimeConfiguration.class)
public interface WcWorkloadServerRuntimeConfiguration
        extends RuntimeConfiguration, WorkloadServerRuntimeConfiguration {}
```

Reply:
Oh I see what you mean, I can just wrap it

Reply:
yes,

```
public interface WcWorkfloadServerRuntimeConfiguration extends RuntimeConfiguration {
  WorkloadServerRuntimeConfiguration workloadConfig();
}
```